manifests/repo/debian.pp

```diff
@@ -70,22 +74,25 @@
     }
   }
 
-  if ($php::globals::php_version == '7.1') {
+  if ($sury and $php::globals::php_version == '7.1') {
     # Required packages for PHP 7.1 repository
     ensure_packages(['lsb-release', 'ca-certificates'], {'ensure' => 'present'})
 
     # Add PHP 7.1 key + repository
     apt::key { 'php::repo::debian-php71':
-      key        => 'DF3D585DB8F0EB658690A554AC0E47584A7A714D',
-      key_source => 'https://packages.sury.org/php/apt.gpg',
+      id     => 'DF3D585DB8F0EB658690A554AC0E47584A7A714D',
+      source => 'https://packages.sury.org/php/apt.gpg',
     }
 
     ::apt::source { 'source_php_71':
-      location    => 'https://packages.sury.org/php/',
-      release     => $::lsbdistcodename,
-      repos       => 'main',
-      include_src => false,
-      require     => [
+      location => 'https://packages.sury.org/php/',
+      release  => $::lsbdistcodename,
```

Comment:
preferred way would be to use the new facts hash instead of a topscope variable. If you want you can change this in another PR, or we just stick with it.